tests/providers/amazon/aws/sensors/test_sqs.py

```diff
@@ -18,7 +18,7 @@
 
 
 import unittest
-from unittest.mock import MagicMock, patch
+import unittest.mock as mock
```

Comment:
I think this will break things. You removed `patch` but it is still required at line 75 and 97.

Wasn't it correct before?

Reply:
I use @mock.patch so it should work. Will see what Travis says :)